lib/shortcode/easy-digital-downloads.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+/**
+ * Shortcode Template File for Easy Digital Downloads
+ */
+
+
+/**
+ * Create our shortcode for Easy Digital Downloads
+ */
+add_action( 'init', 'sandwich_easy_digital_downloads', 11 );
+function sandwich_easy_digital_downloads() {
+
+	// Check if Shortcake exists
+	if ( ! function_exists( 'shortcode_ui_register_for_shortcode' ) ) {
+		return;
+	}
+
+	// Register Shortcake UI for Easy Digital Downloads Download History
+	shortcode_ui_register_for_shortcode( 'download_history', 
+		array(
+			'label' => __( 'Easy Digital Downloads - Download History', 'pbsandwich' ),
```

Comment:
Standardize the labels also. Since we didn't use a dash in the other shortcodes, don't use dashes here. Make this into: *EDD Download History*